plugins/kubernetes/app/assets/javascripts/kubernetes/controllers/kubernetes_create_release_ctrl.js

```diff
@@ -75,10 +76,12 @@ samson.controller('KubernetesCreateReleaseCtrl',
     };
 
     $scope.toggleAll = function() {
-      var newState = !$scope.allToggled();
-      _.each($scope.deploy_groups, function(deploy_group) {
-        deploy_group.selected = newState;
-      });
+      if(!$scope.submitting) {
```

Comment:
Just a suggestion: instead of performing the background computations to check for an ongoing deploy and disable toggling active deploy groups, we could just disable the `Clear All` link and get the same result. An additional benefit would be that the user would actually know that this functionality is disabled.